weldx/geometry.py

```diff
@@ -2696,7 +2706,7 @@ def from_file(file_name: Union[str, Path]) -> SpatialData:
         mesh = meshio.read(file_name)
         triangles = mesh.cells_dict.get("triangle")
 
-        return SpatialData(mesh.points, triangles)
+        return SpatialData(Q_(mesh.points, "mm"), triangles)
```

Comment:
```suggestion
        return SpatialData(Q_(mesh.points, _DEFAULT_LEN_UNIT), triangles)
```

Reply:
I think we shouldn't do this here because we are dealing with files that are not generated by WelDX. So we should stick with the CAD standard (mm). Even though they are currently the same, it would probably better to introduce something like `_CAD_LEN_UNIT`

Reply:
Okay, saw you already handled it in a later commit